healthcheck.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const router = express.Router({});
+
+router.get('/', async (_req, res, _next) => {
+	const healthcheck = {
+		uptime: process.uptime(),
+		message: 'OK',
+		timestamp: Date.now()
+	};
+	try {
+		res.send(healthcheck);
+	} catch (e) {
```

Comment:
When does res.send throw?

Reply:
https://github.com/expressjs/express/blob/5596222f6a6e0eea74f6fb38915b071d98122a2f/lib/response.js#L107

We can catch a potential exception here maybe?